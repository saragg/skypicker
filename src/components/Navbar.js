import React from 'react';
import {Toolbar, ToolbarGroup, ToolbarTitle} from 'material-ui/Toolbar';
import FontIcon from 'material-ui/FontIcon';

const Navbar = (props) => {
  return (
    <Toolbar style={{background: "#fff"}}>
      <ToolbarGroup>
        <ToolbarTitle text="Kiwi Skypicker" />
      </ToolbarGroup>
      <ToolbarGroup>
        <FontIcon className="fa fa-github" />
      </ToolbarGroup>
    </Toolbar>
  )
}

export default Navbar;