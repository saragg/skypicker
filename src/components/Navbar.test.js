import React from 'react';
import Navbar from './Navbar';
import renderer from 'react-test-renderer';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

test('Renders correctly', () => {

  const tree = renderer.create(
    <MuiThemeProvider>
      <Navbar />
    </MuiThemeProvider>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});