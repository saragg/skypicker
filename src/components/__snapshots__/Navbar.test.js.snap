// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className={undefined}
  style={
    Object {
      "WebkitTapHighlightColor": "rgba(0,0,0,0)",
      "background": "#fff",
      "backgroundColor": "rgb(232, 232, 232)",
      "boxSizing": "border-box",
      "display": "flex",
      "height": 56,
      "justifyContent": "space-between",
      "muiPrepared": true,
      "padding": "0px 24px",
    }
  }
>
  <div
    className={undefined}
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "justifyContent": "space-between",
        "marginLeft": undefined,
        "marginRight": undefined,
        "muiPrepared": true,
        "position": "relative",
      }
    }
  >
    <span
      style={
        Object {
          "color": "rgba(0, 0, 0, 0.4)",
          "fontFamily": "Roboto, sans-serif",
          "fontSize": 20,
          "lineHeight": "56px",
          "muiPrepared": true,
          "overflow": "hidden",
          "paddingRight": 16,
          "position": "relative",
          "textOverflow": "ellipsis",
          "whiteSpace": "nowrap",
        }
      }
    >
      Kiwi Skypicker
    </span>
  </div>
  <div
    className={undefined}
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "justifyContent": "space-between",
        "marginLeft": undefined,
        "marginRight": undefined,
        "muiPrepared": true,
        "position": "relative",
      }
    }
  >
    <span
      className="fa fa-github"
      color="rgba(0, 0, 0, 0.4)"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      style={
        Object {
          "color": "rgba(0, 0, 0, 0.4)",
          "cursor": "pointer",
          "display": "inline-block",
          "fontSize": 24,
          "lineHeight": "56px",
          "muiPrepared": true,
          "paddingLeft": 24,
          "position": "relative",
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "userSelect": "none",
        }
      }
    />
  </div>
</div>
`;
